ex1/ex1/Exercises/EnumSchoolSubjects.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ex1.Exercises
+{
+    enum EnumSchoolSubjects
+    {
+        Maths,
+        Physics,
+        Chemistry,
+        Biology,
+        English_literature
```

Comment:
Better to use CamelCase notation here i.e. instead of English_literature => EnglishLiterature